syntax = "proto3";

message Point {
  int32 car = 1;
  int32 lastX = 2;
  int32 lastY = 3;
  int32 nowX = 4;
  int32 nowY = 5;
  int32 goalX =6;
  int32 goalY =7;
}

message Step {
  int32 nextX = 1;
  int32 nextY = 2;
}

service Guide {
  rpc GetNextStep(Point) returns (Step) {}
}
